src/helpers.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if (!function_exists('consume')) {
+    function consume ($closure, $bindingKey = '', array $config = [])
```

Comment:
Wouldn't it be great if it's written like `amqp_consume` `amqp_publish`? Isn't it better being explicit?